htdocs/admin/dolistore/class/PSWebServiceLibrary.class.php

```diff
@@ -222,26 +279,21 @@ public function getVersion()
 	/**
 	 * Load XML from string. Can throw exception
 	 *
-	 * @param 	string 				$response 	String from a CURL response
-	 * @return 	SimpleXMLElement|boolean		status_code, response
+	 * @param string $response String from a CURL response
 	 *
+	 * @return SimpleXMLElement status_code, response
 	 * @throws PrestaShopWebserviceException
 	 */
 	protected function parseXML($response)
 	{
 		if ($response != '') {
 			libxml_clear_errors();
 			libxml_use_internal_errors(true);
-			libxml_disable_entity_loader(true);	// Avoid load of external entities (security problem). Required only if LIBXML_VERSION < 20900
-
-			if (!function_exists('simplexml_load_string')) {
-				throw new PrestaShopWebserviceException('Method simplexml_load_string not available. Your PHP does not support xml.');
-			}
-			$xml = simplexml_load_string($response, 'SimpleXMLElement', LIBXML_NOCDATA|LIBXML_NONET);
+			$xml = simplexml_load_string(trim($response), 'SimpleXMLElement', LIBXML_NOCDATA);
```

Comment:
This introduce a security hole. To fix it we can use both

`libxml_disable_entity_loader(true);`	// Avoid load of external entities (security problem). Required only if LIBXML_VERSION < 20900

and

`$xml = simplexml_load_string(trim($response), 'SimpleXMLElement', LIBXML_NOCDATA|LIBXML_NONET);`